conversions/bytes_to_int.py

```diff
@@ -0,0 +1,59 @@
+def bytes_to_int(bytes_var: bytearray, signed: bool, byteorder: str) -> (int):
+
+    """
```

Comment:
```suggestion
    r"""
```
This leading `r` will make the docstring a `raw` string and then we should be able to shift to single backslashes.

Then we can test with something like...
```
    >>> tests = ((b"\x00\x10", "big", False), (b'\xfc\x00', "big", True),
    ...          (b'\x00\x01', "big", False), (b'\x00\x10', 'little', False))
    >>> all(bytes_to_int(bytes, byteorder, signed=signed) == int.from_bytes(bytes, byteorder, signed=signed)
    ...     for bytes, byteorder, signed in tests)
    True
    >>> bytes_to_int('abc',False,'big')
    Traceback (most recent call last):
        ...
    TypeError: cannot convert 'str' object to bytes
```

Reply:
Extra credit: Exceptions for...
```
>>> bytes_to_int(b'\xfc\x00', 'medium')
>>> bytes_to_int(7, 'big',)
>>> bytes_to_int(7.1, 'little', signed=True)
```